### PR TITLE
QA-235: pipeline: Switch to static checks by golangci-lint

### PR DESCRIPTION
**Pull request by lluiscampos**
Modify the existing .golangci.yml to follow standard and fix goimports
linting issues.

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -32,10 +33,15 @@ linters:
 
 linters-settings:
   gocyclo:
-    min-complexity: 20 # default is 30.
+    # default is 30.
+    min-complexity: 20
+
+  goimports:
+    local-prefixes:
+      "github.com/mendersoftware/deviceconnect"
```
Comment (merlin-northern) [inline .golangci.yml:41]: all very cool, but why it is present here, while absent here: https://github.com/mendersoftware/devicemonitor/pull/69 ?

File: store/mongo/datastore_mongo.go [hunk 2]
```diff
@@ -22,24 +22,23 @@ import (
...
 
 	"github.com/mendersoftware/go-lib-micro/config"
 	"github.com/mendersoftware/go-lib-micro/identity"
+	"github.com/mendersoftware/go-lib-micro/log"
 	"github.com/mendersoftware/go-lib-micro/mongo/migrate"
 	mstore "github.com/mendersoftware/go-lib-micro/store"
+	"github.com/mendersoftware/go-lib-micro/ws"
+	"github.com/mendersoftware/go-lib-micro/ws/shell"
 
+	"github.com/mendersoftware/deviceconnect/app"
```
Comment (merlin-northern) [inline store/mongo/datastore_mongo.go:41]: I bet this is the same as I did here: https://github.com/mendersoftware/deviceconnect/pull/166 :)

**Review by merlin-northern — APPROVED** [review]

File: .golangci.yml (hunk 1 above)
Comment (lluiscampos) [reply .golangci.yml:41]: @merlin-northern It is [there](https://github.com/mendersoftware/devicemonitor/pull/69/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9L40). It was there already.

File: store/mongo/datastore_mongo.go (hunk 2 above)
Comment (lluiscampos) [reply store/mongo/datastore_mongo.go:41]: Maybe, but I will merge first :stuck_out_tongue:

**Comment by lluiscampos** [comment]
Only imports ordering changes, so skipping integration pipeline. Merging.

File: store/mongo/datastore_mongo.go (hunk 2 above)
Comment (merlin-northern) [reply store/mongo/datastore_mongo.go:41]: I saw through your plan immediately. haha

File: .golangci.yml (hunk 1 above)
Comment (merlin-northern) [reply .golangci.yml:41]: oh I see -- I checked but I could not find it. sorry for the noise.

**Pull request merged**